assets/preload/data/introText.txt

```diff
@@ -1,69 +1,56 @@
+crayons are sucky
+winter is bad
+venus is god
 shoutouts to tom fulp--lmao
-Ludum dare--extraordinaire
-cyberzone--coming soon
-love to thriftman--swag
-ultimate rhythm gaming--probably
-dope ass game--playstation magazine
-in loving memory of--henryeyes
-dancin--forever
-funkin--forever
-ritz dx--rest in peace lol
-rate five--pls no blam
-rhythm gaming--ultimate
-game of the year--forever
-you already know--we really out here
-rise and grind--love to luis
-like parappa--but cooler
-album of the year--chuckie finster
-free gitaroo man--with love to wandaboy
-better than geometry dash--fight me robtop
-kiddbrute for president--vote now
-play dead estate--on newgrounds
-this is a god damn prototype--we workin on it okay
-women are real--this is official
-too over exposed--newgrounds cant handle us
-Hatsune Miku--biggest inspiration
-too many people--my head hurts
-newgrounds--forever
-refined taste in music--if i say so myself
-his name isnt keith--dumb eggy lol
-his name isnt evan--silly tiktok
-stream chuckie finster--on spotify
-never forget to--pray to god
-dont play rust--we only funkin
-good bye--my penis
-dababy--biggest inspiration
-fashionably late--but here it is
-yooooooooooo--yooooooooo
-pico funny--pico funny
-updates each friday--on time every time
-shoutouts to mason--for da homies
-bonk--get in the discord call
-kbhgames--steal mods
-this isn't a copy of ke--*vine boom*
-we do a little trolling--h
-sexiest engine--in da block
-he's named evilsk8er--not evilsker
-actual functional--input system
-its nerf--or nothing
-penis--balls even
-fnf is not for kids--go outside
-little herobrine--im cumming in your mom
-what you know about--rolling down in the deep
-oh god i'm white--and i'm on twitter
-i gotta--bucket of chicken
-dream stans--literally scare me
-weeg mod--check it out
-what da dog--doin
-go mango go--go mango go
-go pico yeah yeah--*twerks cutely*
-forever engine--my beloved
-wash your hands--stay safe bro
+dry rocky land
+stream smile and laugh english now
+venus is sussy
```

Comment:
Not only do these not follow the typical intro text format, they're utterly useless changes and wipe away all the previous intro text.